lib/core/text/abstract_text.nit

```diff
@@ -2021,7 +2021,7 @@ end
 # see `alpha_comparator`
 private class AlphaComparator
 	super Comparator
-	redef fun compare(a, b) do return a.to_s <=> b.to_s
+	redef fun compare(a, b) do return a.as(not null).to_s <=> b.as(not null).to_s
```

Comment:
I dont know. this removes warnings an make the code more robust. No existing valid case are changed. You did basically the same thing with `Set::hash` that does not crash on null elements